dev-support/ci/selective_ci_checks.bats

```diff
@@ -368,3 +368,27 @@ load bats-assert/load.bash
   assert_output -p needs-integration-tests=false
   assert_output -p needs-kubernetes-tests=false
 }
+
+@test "LICENSE" {
+  run dev-support/ci/selective_ci_checks.sh a9bb08889c
+
+  assert_output -p 'basic-checks=["rat"]'
+  assert_output -p needs-build=true
+  assert_output -p needs-compile=false
+  assert_output -p needs-compose-tests=false
+  assert_output -p needs-dependency-check=false
+  assert_output -p needs-integration-tests=false
+  assert_output -p needs-kubernetes-tests=false
+}
+
+@test "dependency helper" {
+  run dev-support/ci/selective_ci_checks.sh 47a5671cc5
```

Comment:
This change:  47a5671cc5  just modifies:
*hadoop-ozone/dist/src/main/license/update-jar-report.sh*

That test doesn't seem to have anything to do the licence/notice changes made in this PR.  Why was this test added?

Reply:
The goal of the dependency check is to catch addition/removal of third-party dependencies, which require updating the license file.  It uses `update-jar-report.sh` as a helper script.  So changes to the script are related to license.

Reply:
ah, I should have figured that.  Thanks!